public/components/windowManager.js

```diff
@@ -31,6 +31,9 @@ function createWindow() {
       minHeight: 800,
       center: true,
       title: 'Network Canvas',
+      webPreferences: {
+        nodeIntegration: true,
```

Comment:
We had been attempting to remove the need for nodeIntegration, as per this discussion: https://github.com/codaco/Network-Canvas/issues/472

What are the remaining areas of functionality that require it?